### PR TITLE
Add icons to buttons

### PR DESCRIPTION
**Pull request by dabreegster**
For #134. Now the buttons are more visually distinct:
Screenshot from 2023-05-15 15-06-29

They correctly change color when hovering, same as the text:

https://github.com/acteng/atip/assets/1664407/8c307373-ffb0-426a-a769-766b52cc523b

I picked icons from https://fonts.google.com/icons?selected=Material+Icons+Sharp. There wasn't an obvious choice for most; other ideas very welcome.

CC @Pete-York for review

### CODE REVIEW

File: src/lib/draw/Button.svelte [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  export let changeMode: (m: Mode) => void;
+</script>
+
+<button
+  type="button"
+  on:click={() => changeMode(thisMode)}
+  disabled={mode == thisMode}><img src={icon} alt={label} /> {label}</button
+>
+
+<style>
```
Comment (dabreegster) [inline src/lib/draw/Button.svelte:17]: Lots of this moved from Toolbox

File: src/lib/draw/Button.svelte [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    transition-duration: 0.4s;
+    border: solid 2px black;
+  }
+
+  button:disabled {
+    opacity: 0.6;
+    cursor: auto;
+  }
+
+  /* The SVG icons are defined with black. By default, make them white, using
```
Comment (dabreegster) [inline src/lib/draw/Button.svelte:44]: Alternatively, we could use `<svg>` and use simpler CSS to change color. But then we have to figure out the build system... https://natclark.com/tutorials/svelte-working-with-svgs/ and https://joshuatz.com/posts/2021/using-svg-files-in-svelte/#directly-inlining may be relevant. This way works for now, though

File: src/lib/draw/Button.svelte [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+   * https://codepen.io/sosuke/pen/Pjoqqp to calculate the filter. */
+  img {
+    filter: invert(100%) sepia(0%) saturate(0%) hue-rotate(328deg)
+      brightness(100%) contrast(101%);
+    vertical-align: middle;
```
Comment (dabreegster) [inline src/lib/draw/Button.svelte:49]: Screenshot from 2023-05-15 15-09-55
This isn't what I wanted, but couldn't figure out how to CSS this properly

**Comment by robinlovelace-ate** [comment]
I love these new icons. Subtle, clear, and they even change colour nicely! Who needs a UX designer when you've got software engineers like this ; ) Seriously though the attention to detail is impressive. With more people making suggestions to the UI the rate at which ATIP gains some polish will increase.

**Review by robinlovelace-ate — APPROVED** [review]

**Comment by dabreegster** [comment]
And a trick/reminder for Pete's benefit: this PR is demoable at https://acteng.github.io/atip/icon_buttons/scheme.html?authority=Derby#11.74/52.9147/-1.4704 without cloning and running locally. You can always add `/branch_name/` to make that work, thanks to the magic of https://github.com/acteng/atip/blob/main/.github/workflows/web.yml deploying all branches in the repo

**Pull request merged**